doc/changelog.rst

```diff
@@ -6,6 +6,22 @@ Changes in Version 4.8.0
 
 .. warning:: PyMongo 4.8 drops support for Python 3.7 and PyPy 3.8: Python 3.8+ or PyPy 3.9+ is now required.
 
+Changes in Version 4.7.2
+-------------------------
+
+Version 4.7.2 fixes a bug introduced in 4.7.0:
+
+- Fixed a bug where PyMongo could not be used with the Nuitka compiler.
+
+Issues Resolved
+...............
+
+See the `PyMongo 4.7.2 release notes in JIRA`_ for the list of resolved issues
```

Comment:
There's something funny with these links. For one, they not automatically take you to them by clicking in github. But more importantly, when I paste the link into my browser, I don't see things rendered as html. I see raw tags, like <h2>.

Reply:
We're moving away from this changelog toward the "What's New" page on the internal docs site.

Reply:
So this is how it's supposed to look?

Reply:
🤷🏼‍♂️